LambdaEngine/Include/ECS/ECSCore.h

```diff
@@ -13,6 +13,14 @@ namespace LambdaEngine
 	class EntitySubscriber;
 	class RegularWorker;
 
+	// EntitySerializationHeader is written to the beginning of an entity serialization
+	struct EntitySerializationHeader
+	{
+		uint32 TotalSerializationSize;
+		Entity Entity;
+		uint32 ComponentCount;
+	};
```

Comment:
Same as the last comment
```suggestion
#pragma pack(push, 1)
	struct EntitySerializationHeader
	{
		uint32 TotalSerializationSize;
		Entity Entity;
		uint32 ComponentCount;
	};
#pragma pack(pop)	
```